snap/snapcraft.yaml

```diff
@@ -203,6 +203,10 @@ parts:
           config="arm64_kata_kvm_${x_version}"
         ;;
 
+        "s390x")
+          config="s390_kata_kvm_${x_version}"
```

Comment:
kernel version 4.14 doesn't exist  https://github.com/kata-containers/packaging/tree/master/kernel/configs

Reply:
Yes I'm aware of that. AKAIK work is being done to move to 4.19 series on all architectures. s390x moved/upgraded to 4.19 a while ago. Obviously after the PR for updating 4.19 is merged, `runtime/versions.yaml` is updated with 4.19, then I can bring back the s390x CI builder to life.